harmoni_detectors/harmoni_stt/config/direct_configuration.yaml

```diff
@@ -0,0 +1,17 @@
+w2l:
+    default_param:
+        model_path: /root/model/w2l/
+        w2l_bin: /root/wav2letter/build/inference/inference/examples/simple_streaming_asr_example
+        subscriber_id: "default"
+        audio_format_width: 2
+        chunk_size: 1024
+        total_channels: 1
+        audio_rate: 16000
+        silence_limit_seconds: 1
+        previous_audio_seconds: 2
+        total_silence_samples: 100
+        silence_threshold: 1000
+        device_name: default
+        loudest_sound_value: 5.5
```

Comment:
Do we have any calibration tool for at least getting a baseline for these values so people can be informed about the settings? Might be something to put in the readme if it's not already there.

Reply:
There is a function for livesetting it, but honestly it could use some work. I would make this an issue